tutorials/scripting/c_sharp/c_sharp_variant.rst

```diff
@@ -0,0 +1,156 @@
+.. _doc_c_sharp_variant:
+
+C# Variant
+==========
+
+For a detailed explanation of Variant in general, see the :ref:`doc_variant_class` section in the core and modules tutorial.
```

Comment:
Is this the right Variant page to link to? I noticed there's https://docs.godotengine.org/en/latest/classes/class_variant.html and https://docs.godotengine.org/en/latest/contributing/development/core_and_modules/variant_class.html.

Reply:
This should link to the latter.

Reply:
I think it should link to the former. The class reference page already links to the latter.

Reply:
Though I guess, it doesn't matter much. Both pages are kind of bare.

Reply:
Currently it links to the latter, I think I'm leaning towards linking to the former instead like @YuriSizov suggests.

Reply:
Hard to make a meaningful distinction between the two :D But if I have to, former would be for more user-oriented explanation, and latter for more low-level dev explanation. So for general user documentation, class references should be preferred, unless this part specifically wants to point to the implementation detail.

Keep in mind the text should also be adjusted if you change the link.

Reply:
The line below still points to the engine doc. Which would be very confusing to the user, since they would appear visually the same :P

Reply:
I think it makes more sense to link to the class reference then. I also opened https://github.com/godotengine/godot/pull/72721 to update it though because the C# examples are outdated.